src/libstore/gc-store-api.cc

```diff
@@ -0,0 +1,13 @@
+#include "gc-store-api.hh"
```

Comment:
I would just call this `gc-store.hh` and `gc-store.cc`. API is redundant since every header is an API...

(Yes, `store-api.hh` is also a bad name.)

Reply:
:) yes that was my thought too but didn't want to buck the pattern unless you agreed!